src/installer/pkg/sfx/installers/dotnet-host.proj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
     <SkipBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</SkipBuild>
-    <GenerateInstallers>true</GenerateInstallers>
```

Comment:
I don't think we can remove this without making it difficult/impossible to build these installers individually.

Reply:
I removed this because I thought they would pull the setting from the `../../Directory.Build.props` -- any reason that doesn't work?

Reply:
Hmm, seems like it didn't build the wixproj locally with my change, but I can't figure out why.

Reply:
I didn't realize it was set in the parent Directory.Build.props.